tests/ui/test_validation_ui.py

```diff
@@ -94,3 +94,18 @@ def test_odf_cephblockpool_compression_status(self, setup_ui_class):
             f"'Compression status' from ocs-storagecluster-cephblockpool = {compression_statuses[1]}\n"
             f"Expected: {compression_status_expected}"
         )
+
+    @ui
+    @pytest.mark.bugzilla("1994584")
+    def test_odf_operator_in_installed_opearator(self, setup_ui_class):
```

Comment:
1. Operator spelling is incorrect
2. I think test name should be changed to something like `test_ocs_operator_isn't_present` or something like this to make it clear what's the expectation from this test or the BZ

Reply:
1. operator spelling seems correct.
2. changed the function name to suggested one.